plugins/apps/functions.go

```diff
@@ -34,6 +35,11 @@ func createApp(appName string) error {
 
 	common.LogInfo1Quiet(fmt.Sprintf("Creating %s...", appName))
 	os.MkdirAll(common.AppRoot(appName), 0755)
+	err := os.WriteFile(fmt.Sprintf("%v/GENESIS", common.AppRoot(appName)), 
```

Comment:
Can we use the property system for this instead? I don't want to litter the repo directory with more files.

Reply:
thanks for the quick review @josegonzalez 

I tried looking for another way to tell when a folder was created but it seems like Linux only provides `atime`, `mtime` and `ctime`. Any pointers on which property system should be used?

Reply:
I meant more like [these functions](https://github.com/dokku/dokku/blob/master/plugins/common/properties.go). They basically provide a k/v interface for storing information.

Reply:
I've made the changes @josegonzalez